docs/docs/guides/contributing-components.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Contributing Components
+sidebar_label: Contributing Components
+---
+
+:::warning
+
+This functionality is under development. With these docs, we aim to elicit feedback from our community (you!) and focus our development efforts. Currently, we do not offer support for 3rd party components, but we are working toward supporting this.
+
+:::
+
+`Airy Core` is fundamentally a collection of components. When you [install `Airy Core`](../getting-started/installation/introduction.md), you install a set of _core_ components which create the minimum infrastructure to install other components and begin using the platform.
```

Comment:
`Airy Core` is not really how we market the open source. Let's just call it `Airy` and mention that it shipds with a few components out-of-the-box.

Reply:
Sounds good. Although there are a ton of references to Airy Core in the docs, so we might need a larger fix. For now I will refer to it as Airy.